apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue

```diff
@@ -212,15 +212,36 @@ const initSelectedMenuItems = async (): Promise<SelectDropdownMenuItem[]> => {
 const updateWidgetOptionsBySelected = (selected?: SelectDropdownMenuItem[]) => {
     const propertyName = props.propertyName;
     const widgetOptions = { ...widgetFormState.widgetOptions };
+    const dataName = propertyName.replace('filters.', '');
+
+    // add case
     if (selected?.length) {
         if (state.schemaProperty?.selection_type === 'SINGLE') {
-            widgetOptions[propertyName] = selected[0].name;
+            if (propertyName.includes('filters')) {
+                widgetOptions.filters = {
+                    ...widgetOptions.filters,
+                    [dataName]: selected[0].name,
+                };
+            } else {
+                widgetOptions[propertyName] = selected[0].name;
+            }
+        } else if (propertyName.includes('filters')) {
+            widgetOptions.filters = {
+                ...widgetOptions.filters,
+                [dataName]: selected.map((item) => item.name),
+            };
         } else {
             widgetOptions[propertyName] = selected.map((item) => item.name);
         }
+    }
+
+    // delete case
+    if (propertyName.includes('filters')) {
```

Comment:
suggestion: I suggest below: 

```suggestion
    if (propertyName.startsWith('filters.')) {
```